source/iface/context.rst

```diff
@@ -29,29 +34,29 @@ API to interact with a group of devices associated with a platform.
 
 ::
 
-  explicit context(const property_list& propList = {});
+  explicit context(const sycl::property_list& propList = {});
```

Comment:
Should we here add also explicit sycl::context?

Reply:
Yes, missed it with propList, thanks a lot!

Not sure we should add sycl:: prefix to constructors, but we can discuss it...

Reply:
Yes, missed it with propList, thanks a lot!

Not sure we should add sycl:: prefix to constructors, but we can discuss it...